SamplesDashboard/SamplesDashboard/ClientApp/src/views/Repositories.tsx

```diff
@@ -324,16 +319,24 @@ function compare(a: any, b: any, isSortedDescending?: boolean) {
 function checkStatus(status: number) {
     switch (status) {
         case 0:
-            return <span><FontIcon iconName="StatusCircleQuestionMark" className={classNames.blue} /> Unknown </span>;
+            return <TooltipHost content="Unknown" id={'Unknown'}>
+                <span><FontIcon iconName="StatusCircleQuestionMark" className={classNames.blue} /> Unknown </span>
+            </TooltipHost>;
 
         case 1:
-            return <span><FontIcon iconName="CompletedSolid" className={classNames.green} /> Up To Date </span>;
+            return <TooltipHost content="This repository has dependencies that are up to date" id={'UptoDate'}>
```

Comment:
>This repository has dependencies that are up to date [](start = 41, length = 52)

i think the intent is to say that all dependencies are up to date otherwise we wouldn't be in this case.  The sentence sounds like some dependencies are up to date but not necesssarily all.

Reply:
Yes, this is true. I will correct the sentence used.